database/database.go

```diff
@@ -27,7 +27,7 @@ type Cell struct {
 
 // Connect to a (for now) MySQL database with the provided DSN
 func Connect(dsn string) (*sql.DB, error) {
-	conn, err := sql.Open("mysql", dsn)
+	conn, err := sql.Open("mysql", dsn+"?parseTime=true")
```

Comment:
`fmt.Sprintf("%s?parseTime=true", dsn)`

Reply:
Here, I think it's just a matter of style. And the code is probably more readable with `+` than with `fmt.Sprintf`. Also, in terms of efficiency [concatenation is faster](https://gist.github.com/dtjm/c6ebc86abe7515c988ec). Although, to be fair, I can't think of a use case where this efficiency becomes important.

I'd use `fmt.Sprintf` for error messages or something like that where searching the code for the string is important.